feature-utils/poly-look/src/css/spacing.css

```diff
@@ -0,0 +1,27 @@
+:root {
+/* Spacing units for margins, padding, grid, position (top, bottom, right, left) */
+  --pl-space1x: 4px;
+  --pl-space2x: calc(var(--pl-space1x) * 2);
+  --pl-space3x: calc(var(--pl-space1x) * 3);
+  --pl-space4x: calc(var(--pl-space1x) * 4);
+  --pl-space5x: calc(var(--pl-space1x) * 5);
+  --pl-space6x: calc(var(--pl-space1x) * 6);
+  --pl-space7x: calc(var(--pl-space1x) * 7);
+  --pl-space8x: calc(var(--pl-space1x) * 8);
+  --pl-space9x: calc(var(--pl-space1x) * 9);
+  --pl-space10x: calc(var(--pl-space1x) * 10);
+
+/* Border radius */
+  --pl-radius1x: 4px;
+  --pl-radius2x: calc(var(--pl-radius1x) * 2);
+  --pl-radius3x: calc(var(--pl-radius1x) * 3);
+  --pl-radius4x: calc(var(--pl-radius1x) * 4);
+  --pl-radius5x: calc(var(--pl-radius1x) * 5);
+  --pl-radius6x: calc(var(--pl-radius1x) * 6);
+  --pl-radius7x: calc(var(--pl-radius1x) * 7);
+  --pl-radius8x: calc(var(--pl-radius1x) * 8);
+
```

Comment:
@victorjanin I would like to implement some of this variables as an example in this PR, but please, let me know first if the naming and idea is ok.

Reply:
I would write it this way
`--pl-radius-1x`

Also I would separate _border-radius_ in another file maybe. 
Either we have all foundation tokens in one file, or we separate them. Because border radius doesn't belong in a spacings file imho.

Reply:
I assume I should also change the name of the other variables to `--pl-space-1x`, correct?

Reply:
Yes, the idea is to separate all values that could change. 
`pl` is the systems' name
`radius` or `space` is your foundation (part of the base) 
`1x` is your scale
![Intersect](https://user-images.githubusercontent.com/7642309/177518565-adf11672-fcbd-4d57-a627-29e73b01a0c7.png)